superset/assets/javascripts/explorev2/validators.js

```diff
@@ -0,0 +1,32 @@
+/* Reusable validator functions used in controls definitions
+ *
+ * validator functions receive the v and the configuration of the field
+ * as arguments and return something that evals to false if v is valid,
+ * and an error message if not valid.
+ * */
+
+export function numeric(v) {
```

Comment:
I don't think `numeric` is confusing, `float` might sound like it has to have decimals, though either work pretty much just as well I think.